src/pages/accounts/Accounts.tsx

```diff
@@ -1,24 +1,21 @@
 import { Suspense, lazy } from 'react';
-import { useLocation } from 'react-router-dom';
+import { Route, Routes } from 'react-router-dom';
 import { AccountsLoader } from '../../components/Loaders/Loaders';
-import routes from '../../utils/routes';
-import Navigation from '../../components/navigation/nav';
-import './login/style.scss';
 
 const Tfa = lazy(() => import('./tfa/Tfa'));
 const Signup = lazy(() => import('./signup/signup'));
 const Login = lazy(() => import('./login/Login'));
 
 export default function Accounts() {
-  const location = useLocation();
-  const { pathname } = location;
   return (
     <div className="account__container" data-testid="accounts">
-      <Navigation />
       <Suspense fallback={<AccountsLoader />}>
-        {routes.authenticate === pathname && <Tfa />}
-        {routes.signup === pathname && <Signup />}
-        {routes.login === pathname && <Login />}
+        <Routes>
+          <Route path="/authenticate" element={<Tfa />}></Route>
+          <Route path="/signup" element={<Signup />}></Route>
+          <Route path="/authenticate" element={<Tfa />} />
```

Comment:
line 16, and line 14 are totally the same
`  <Route path="/authenticate" element={<Tfa />}></Route>` and this :   ` <Route path="/authenticate" element={<Tfa />} />`